### PR TITLE
Adds in the Jolly Jug

### PR DESCRIPTION
**Pull request by shazbot194**
Due to my master exploding, I had to remake a couple of my PRs. This pull adds in the Jolly Jug, the clown version of a bottle of nothing. The clown spawns with it and has a backup in their locker, just like the mime, in addition it spawns with 100u of banana juice.

et8nu6v

:cl: Shazbot-coding, Driker-Sprites
Add: Gives the clown their own jug
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -173,6 +173,12 @@
 	icon_state = "bottleofnothing"
 	list_reagents = list("nothing" = 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/bottleofbanana
+	name = "Jolly Jug"
```
Comment (Citinited) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: make this lower-case

Comment (shazbot194) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: Currently, all bottles, including the Bottle of Nothing is capitalized, making it lower-case would make it inconsistent.

Comment (Citinited) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: I suppose, though I dislike it...

Comment (shazbot194) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: If it makes you feel better, it is more or less being used as a proper noun, more so then bottle of nothing, and would be capitalized anyway.

**Comment by Kyep** [comment]
This effectively starts the clown off with the ability to heal up to 100 pts of brute *and* 100 pts of burn, divisible into any number of shots, by drinking this. While I understand that clowns need more healing than most, starting them off with an item that enables them to heal so much at will is a fairly big buff to clowns. Knowing that clowns have this also encourages people to beat up on clowns, because they know that the clown can just drink their juice to heal themselves afterwards.

**Comment by Vivalas** [comment]
Doesn't the mime already have that ability though?

**Comment by The-Driker** [comment]
@ViVala

Correct. This was also noted in the original PR that got deleted.

**Comment by shazbot194** [comment]
Not to mention that you can just get banana juice from the bartender super easily anyway, they don't even need a multi tool to make it. Not to mention that saline is a much better healer then this as from what I remember this is a one to one heal that is very slow. And I think our community is mature enough to not just beat someone up because they might be able to heal, but rather beat up the clown for slipping them for the 10th time.

**Comment by taukausanake** [comment]
New meta: beat the clown with his own jug and then force feed him the contents to heal before you start beating him again.

Offtopic: am I the only one who is a little turned off that the bartender does not need to depend on ANY assistance from Botany to make drinks despite juicing fruit being a thing?

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: Look at all the other names in this file; they're not lower case.

**Comment by The-Driker** [comment]
I'd say this is probably good to merge. Only thing I see people disagreeing on at this point is upper and lower case letters.

**Pull request merged**